api/groups/rawGroup.go

```diff
@@ -0,0 +1,537 @@
+package groups
```

Comment:
if the API path was renamed from `raw` to `internal`, I suggest that you rename this files and other namings as well